modules/light-clients/08-wasm/internal/ibcwasm/wasm_no_cgo.go

```diff
@@ -0,0 +1,5 @@
+//go:build !cgo
+
+package ibcwasm
+
+var _ WasmEngine = (WasmEngine)(nil)
```

Comment:
this is basically a no-op right? Do we even need this file?

Reply:
yeah I think we can delete it